### DISCUSSION TITLE
[DÚVIDA] Layout mais usado?

### DISCUSSION THREAD

**Discussion by orochasamuel** [orochasamuel/fiscalbr-net#89] [General]
Galera,

Qual o layout mais usado por vocês? É a EFD Fiscal? EFD Contribuições? Ambos? Tem alguém usando em produção a ECD ou ECF?

No segundo trimestre de 2023 quero me concentrar em estabilizar o layout mais usado pela comunidade, para depois ir atualizando os demais.

Em todos os módulos do SPED disponíveis na lib há uma série de funcionalidades e correções (e implementações) a serem feitas, a exemplo dos enumeradores que ainda não foram incluídos ou mesmo que já estão incluídos mas não estão sendo referenciados nos blocos.

@adrbarros
@adrianotrentim
@davi-barboza
@eduarda80
@ernanicl
@esistemsistemas
@hwsistemas-team
@marcianobandeira
@marcosgerene
@nerd-working
@PGama-Rodrigo
@RenatoGuarilhaCMCorp
@vitor85
@williamoura
@leandrossantos
@lvitorbh
@guilhermegriloamorim
@pasturchak
@leandrovip
@FabioDlink
@eduprog
@candinho1987
@EttorePaludeto
@jaina-egler
@Dufayel
@kslagerstrom
@alexandredutra
@alxmath
@leandrosantoslss
@bruno-crr

Vida longa e próspera 🖖

**Comment by adrbarros** [comment]
Olá, utilizamos a lib para geração do EFD ICMS/IPI e EFD CONTRIBUIÇÕES.
Utilizamos também para gerar Sintegra para apenas 1 Cliente no Estado de MG

**Reply by alxmath** [reply]
Olá, aqui estamos desenvolvendo um sistema para EFD ICMS IPI e Contribuições.

**Comment by marcianobandeira** [comment]
Olá, EFD Icms Ipi e EFD Contribuições por enquanto

**Comment by WashingtonAutocom** [comment]
Olá por aqui usamos ambos EFD Fiscal e EFD Contribuições, também geramos Sintegra

**Comment by vitor85** [comment]
Olá utilizamos muito o EFD Fiscal e EFD Contribuição.

Em qui., 30 de mar. de 2023 às 23:24, Washington Reis <
***@***.***> escreveu:

> Olá por aqui usamos ambos EFD Fiscal e EFD Contribuições, também geramos
> Sintegra
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orochasamuel/fiscalbr-net/discussions/89#discussioncomment-5484172>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKHUPHU34R77YE2EBQ4APKTW6Y57BANCNFSM6AAAAAAWNJXKFE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
Atenciosamente,

Vitor Rodrigues
(28) 99926-1565
vitor_rodrigues_pereira

**Comment by candinho-silveira** [comment]
Estou desenvolvendo um aplicativo que usara efd fiscal e contribuições

**Comment by marcosgerene** [comment]
Uso Fiscal e Contribuições.

Muito mais o Fiscal.

**Comment by davi-barboza** [comment]
Atualmente utilizo mais o fiscal, mas estou na fase de implementação do Contribuições.

**Comment by AndersonQS** [comment]
utilizo mais o efdfiscal, estou começando a usar o ecf quando acabar ira pra produção
